packages/datetime/src/dateRangePicker.tsx

```diff
@@ -554,7 +554,8 @@ export class DateRangePicker
 
     private updateRightView(rightView: MonthAndYear) {
         let leftView = this.state.leftView.clone();
-        if (!rightView.isBefore(rightView)) {
+        // a bug here
+        if (!leftView.isBefore(rightView)) {
```

Comment:
@leebyp the original logic here does look funky. Can you take a look and verify this?

Also @chux0519, you can remove the `// a bug here` comment, won't be necessary or helpful after the PR merges.